docs/design/Colors.stories.mdx

```diff
@@ -295,129 +296,8 @@ caution, it’s _bright!_
-      "--color-base-pink--700",
-      "--color-base-pink--800",
-      "--color-base-pink--900",
-      "--color-base-pink--1000",
-      "--color-base-orange--100",
-      "--color-base-orange--200",
-      "--color-base-orange--300",
-      "--color-base-orange--400",
-      "--color-base-orange--500",
-      "--color-base-orange--600",
-      "--color-base-orange--700",
-      "--color-base-orange--800",
-      "--color-base-orange--900",
-      "--color-base-orange--1000",
-      "--color-base-yellow--100",
-      "--color-base-yellow--200",
-      "--color-base-yellow--300",
-      "--color-base-yellow--400",
-      "--color-base-yellow--500",
-      "--color-base-yellow--600",
-      "--color-base-yellow--700",
-      "--color-base-yellow--800",
-      "--color-base-yellow--900",
-      "--color-base-yellow--1000",
-      "--color-base-lime--100",
-      "--color-base-lime--200",
-      "--color-base-lime--300",
-      "--color-base-lime--400",
-      "--color-base-lime--500",
-      "--color-base-lime--600",
-      "--color-base-lime--700",
-      "--color-base-lime--800",
-      "--color-base-lime--900",
-      "--color-base-lime--1000",
-      "--color-base-green--100",
-      "--color-base-green--200",
-      "--color-base-green--300",
-      "--color-base-green--400",
-      "--color-base-green--500",
-      "--color-base-green--600",
-      "--color-base-green--700",
-      "--color-base-green--800",
-      "--color-base-green--900",
-      "--color-base-green--1000",
-      "--color-base-teal--100",
-      "--color-base-teal--200",
-      "--color-base-teal--300",
-      "--color-base-teal--400",
-      "--color-base-teal--500",
-      "--color-base-teal--600",
-      "--color-base-teal--700",
-      "--color-base-teal--800",
-      "--color-base-teal--900",
-      "--color-base-teal--1000",
-      "--color-base-lightBlue--100",
-      "--color-base-lightBlue--200",
-      "--color-base-lightBlue--300",
-      "--color-base-lightBlue--400",
-      "--color-base-lightBlue--500",
-      "--color-base-lightBlue--600",
-      "--color-base-lightBlue--700",
-      "--color-base-lightBlue--800",
-      "--color-base-lightBlue--900",
-      "--color-base-lightBlue--1000",
-      "--color-base-purple--100",
-      "--color-base-purple--200",
-      "--color-base-purple--300",
-      "--color-base-purple--400",
-      "--color-base-purple--500",
-      "--color-base-purple--600",
-      "--color-base-purple--700",
-      "--color-base-purple--800",
-      "--color-base-purple--900",
-      "--color-base-purple--1000",
-      "--color-base-blue--100",
-      "--color-base-blue--200",
-      "--color-base-blue--300",
-      "--color-base-blue--400",
-      "--color-base-blue--500",
-      "--color-base-blue--600",
-      "--color-base-blue--700",
-      "--color-base-blue--800",
-      "--color-base-blue--900",
-      "--color-base-blue--1000",
-      "--color-base-white",
-      "--color-base-black",
-    ]}
+    colors={Object.keys(tokens)
```

Comment:
If there is a better place to grab these colors than `token` that requires less parsing please let me know

Reply:
I think this is the best place to grab them. The other things I could find didn't include the base" variants of things